### PR TITLE
Support openrc (gentoo linux).

### PR DESCRIPTION
**Pull request by s1061123**
Please always submit pull requests on the development branch.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | https://github.com/pm2-hive/pm2-hive.github.io/pulls

*Please update this template with something that matches your PR*

### CODE REVIEW

**Comment by s1061123** [comment]
Hi Gurus,

This PR is to support Gentoo linux that uses openrc, instead of systemd/upstart.
Could you please review this PR?

Regards,
Tomo

File: lib/API/Startup.js [hunk 1]
```diff
@@ -115,6 +116,22 @@ module.exports = function(CLI) {
...
       break;
+    case 'openrc':
+      Common.printOut(cst.PREFIX_MSG + 'Disabling and deleting old startup system');
+      commands = [
+	'/etc/init.d/pm2 stop',
+        'rc-update delete pm2 default',
+        'rm /etc/init.d/pm2'
+      ];
+      break;
+    case 'oldsystem':
```
Comment (vmarchaud) [inline lib/API/Startup.js]: Its already just below, please remove this one.

Comment (s1061123) [inline lib/API/Startup.js]: Thanks. I've fixed.

**Comment by vmarchaud** [comment]
Thanks, it will be merged & published soon

**Comment by vmarchaud** [comment]
Can't make it work on Gentoo v2.2, what is your version ?

**Comment by s1061123** [comment]
Now I use gentoo 2.3 as openrc-0.22.4 as following:
```
tom@kagaribi $ lsb_release -a
LSB Version:    n/a
Distributor ID: Gentoo
Description:    Gentoo Base System release 2.3
Release:        2.3
Codename:       n/a
tom@kagaribi $ equery l '*openrc*'
 * Searching for *openrc* ...
[IP-] [  ] sys-apps/openrc-0.22.4:0
tom@kagaribi $
```

Please let me know if you need more info.

**Comment by vmarchaud** [comment]
Okay, guess it won't work with gentoo 2.2 so, merging anyway.

**Pull request merged**